#
# [xchat 2.6.8 PATCH] Fixes a possible crash when clicking
# on "Private Message" in the 'Advanced > Text Events' window.
#
--- xchat-2.6.8/src/common/text.c	1 Oct 2006 10:31:05 -0000	1.83
+++ xchat-2.6.8p1/src/common/text.c	4 Nov 2006 07:32:54 -0000
@@ -641,6 +641,7 @@
 static char * const pevt_privmsg_help[] = {
 	N_("Nickname"),
 	N_("The message"),
+	N_("Identified text")
 };
 
 static char * const pevt_changenick_help[] = {
@@ -705,12 +706,6 @@
 	N_("The message"),
 };
 
-static char * const pevt_dprivmsg_help[] = {
-	N_("Nickname"),
-	N_("The message"),
-	N_("Identified text"),
-};
-
 static char * const pevt_uchangenick_help[] = {
 	N_("Old nickname"),
 	N_("New nickname"),
--- xchat-2.6.8/src/common/textevents.h	1 Oct 2006 10:31:09 -0000	1.34
+++ xchat-2.6.8p1/src/common/textevents.h	4 Nov 2006 07:32:56 -0000
@@ -302,7 +302,7 @@
 {"Private Message", pevt_privmsg_help, 131, 
 "%C28*%C29$3$1%C28*$t%O$2"},
 
-{"Private Message to Dialog", pevt_dprivmsg_help, 131, 
+{"Private Message to Dialog", pevt_privmsg_help, 131, 
 "%C18%H<%H$3$1%H>%H%O$t$2"},
 
 {"Process Already Running", pevt_generic_none_help, 0, 
@@ -314,7 +314,7 @@
 {"Raw Modes", pevt_rawmodes_help, 2, 
 N_("%C22*%O$t$1 sets modes%B %C30[%O$2%B%C30]")},
 
-{"Receive Wallops", pevt_dprivmsg_help, 2, 
+{"Receive Wallops", pevt_privmsg_help, 2, 
 N_("%C28-%C29$1/Wallops%C28-%O$t$2")},
 
 {"Resolving User", pevt_resolvinguser_help, 2, 
--- xchat-2.6.8/src/common/textevents.in	1 Oct 2006 10:31:09 -0000	1.33
+++ xchat-2.6.8p1/src/common/textevents.in	4 Nov 2006 07:32:56 -0000
@@ -600,7 +600,7 @@
 
 Private Message to Dialog
 XP_TE_DPRIVMSG
-pevt_dprivmsg_help
+pevt_privmsg_help
 %C18%H<%H$3$1%H>%H%O$t$2
 n3
 
@@ -624,7 +624,7 @@
 
 Receive Wallops
 XP_TE_WALLOPS
-pevt_dprivmsg_help
+pevt_privmsg_help
 %C28-%C29$1/Wallops%C28-%O$t$2
 2
 
